src/main/java/com/three_tech_solutions/slot_app/controllers/responses/StudentMonthlyFeeResponseDto.java

```diff
@@ -0,0 +1,13 @@
+package com.three_tech_solutions.slot_app.controllers.responses;
+
+import com.three_tech_solutions.slot_app.data.enums.MonthlyFeeStatus;
+
+import java.time.LocalDate;
+
+public record StudentMonthlyFeeResponseDto(
+        int number,
+        String month,
+        LocalDate expirationDate,
+        double amount,
+        MonthlyFeeStatus status
+) {}
```

Comment:
Deberíamos devolver el ID de la cuota, ya que el frontend debería usarlo para poder decir "pago esta cuota en particular"


```suggestion
public record StudentMonthlyFeeResponseDto(
        UUID id,
        int number,
        String month,
        LocalDate expirationDate,
        double amount,
        MonthlyFeeStatus status
) {}
```